### DISCUSSION TITLE
<title> I accedently made my github repo public and someone forked my repo

### DISCUSSION THREAD

**Discussion by pishabh625** [community/community#39765] [Repositories] [answered]
### Select Topic Area

Question

### Body

I accedently made my github repo public and someone forked it. how do get my repo back.
because it contains all my private project information...
please help me.....

**Comment by david-engelmann** [comment, marked as answer]
So these are the possible scenarios.

```
Deleting a private repository will delete all of its forks.
...
If original private repo deleted: all of its forks will be deleted .
If original public repo deleted: its forks will not be deleted .
If public repo is converted to private, and deleted the original: Forked repo will not be deleted.
```
Unfortunately, if you fall into the last category, you may have issues getting the fork deleted. If deleting the repository doesn't work, you should reach out to support@github.com, they might not be able to help but its worth a try!

**Reply by pishabh625** [reply]
Thanks for helping me.... 😃
